src/components/lists/items/CallAssignmentListItem.jsx

```diff
@@ -41,48 +44,104 @@ export default class CallAssignmentListItem extends React.Component {
 
     render() {
         let assignment = this.props.data;
+        let progressSum = 0;
+        let callsStats = null;
+        let reachedStats = null;
         let targetStats = null;
         let goalStats = null;
-        let participantList = null;
+        let participantIndicator = null;
+        const assignmentDateStart = new Date(assignment.start_date);
+        const assignmentDateEnd = new Date(assignment.end_date);
+        const inPast = (assignmentDateEnd < (new Date()) ? true : false);
+
+        const classNames = cx({
+            'CallAssignmentListItem': true,
+            'past': inPast
+        });
+
+        const assignmentDateSpan =
+            assignmentDateStart.getDate()
+            + "/" + assignmentDateStart.getMonth()
```

Comment:
Use the date formatting utility on `Date` (added by the sugar-js library) instead, i.e. something like `assignmentDateStart.format('{d}/{M}, {yyyy}')`.